packages/forms-web-app/src/views/layouts/multi-file-upload.njk

```diff
@@ -61,7 +61,8 @@
       {% if body %}
          <p class="govuk-body">{{body}}</p>
       {% endif %}
-      
+      {% block optional_html %}
+      {% endblock optional_html %}
```

Comment:
Don't think we need this block

Reply:
I think we do because if I remove it it breaks the unit tests

Reply:
I don't think we should be unit testing the nun jucks templates, we're using the nun jucks rendering engine which is outside of our control. If that doesn't render what we're expecting then it's either an issue with nun jucks or the values we're supplying, which is tested in other places.
The rendered UI will be tested as part of acceptance as well.

Reply:
I shall remove the unit tests.